platform/platform_tests/utm_mgrs_utils_tests.cpp

```diff
@@ -0,0 +1,70 @@
+#include "testing/testing.hpp"
+
+#include "platform/utm_mgrs_utils.hpp"
+
+#include <string>
+
+using namespace utm_mgrs_utils;
+
+#define OPT(a) std::make_optional(a)
```

Comment:
Well, better to make shortcut function ```std::optional<std::string> OPT(std::string s) { return s; }```

But, I'd prefer to avoid ```optional<string>``` if we have an empty string, no?